content/source/docs/enterprise/api/admin/settings.html.md

```diff
@@ -0,0 +1,485 @@
+---
+layout: enterprise2
+page_title: "Private Terraform Enterprise Settings - API Docs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-api-admin-settings"
+---
+
+# Private Terraform Enterprise Settings API
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+-> **Note**: These endpoints are only available in Private Terraform instances and only accessible by site administrators.
+
+## List General Settings
+`GET /api/v2/admin/general-settings`
+
+Status  | Response                                         | Reason
+--------|--------------------------------------------------|-------
+[200][] | [JSON API document][] (`type: "general-settings"`) | Successfully listed General settings
+[404][] | [JSON API error object][]                    | User unauthorized to perform action
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[JSON API document]: https://www.terraform.io/docs/enterprise/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request GET \
+  https://app.terraform.io/api/v2/admin/general-settings
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "general",
+    "type": "general-settings",
+    "attributes": {
+      "support-email-address": "support@example.com",
+      "security-email-address": "security@example.com",
+      "limit-user-organization-creation": true
+    }
+  }
+}
+```
+
+## Update General Settings
+`PATCH /api/v2/admin/general-settings`
+
+Status  | Response                                     | Reason
+--------|----------------------------------------------|----------
+[200][] | [JSON API document][] (`type: "general-settings"`) | Successfully updated the General settings
+[404][] | [JSON API error object][]                    | User unauthorized to perform action
+[422][] | [JSON API error object][]                    | Malformed request body (missing attributes, wrong types, etc.)
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+[JSON API document]: https://www.terraform.io/docs/enterprise/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+### Request Body
+
+This PATCH endpoint requires a JSON object with the following properties as a request payload.
```

Comment:
@nfagerlund's update for SAML and SMTP indicated that attributes not included retain their previous values there. I'm assuming that's also true here?